src/main/java/Duke.java

```diff
@@ -15,20 +15,14 @@ public static void main(String[] args) {
             if(reply.equalsIgnoreCase("LIST")){
                 taskManager.printTasks();
                 String choice = taskManager.taskChoice();
-                while(choice=="ok"){
+                while(choice=="finishedLoop"){
                     choice = taskManager.taskChoice();
                 }
                 reply = choice;
                 continue;
             }
 
-            //save it into the task list
-            Task newTask = new Task(reply);
-            taskManager.addTask(newTask);
-
-            System.out.println("____________________________________________________________");
-            System.out.println("added: "+reply);
-            System.out.println("____________________________________________________________");
+            taskManager.addTask(reply);
 
             reply = input.nextLine();
```

Comment:
Perhaps consider following the same level of abstraction at this level (more higher level functions?)

Reply:
Thank you for your suggestions! I will refactor my code in the next few days^ ^